interview-questions.md

```diff
@@ -4,39 +4,44 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create, Read, Update and Delete
 
   Researched answer:
 
+CRUD          HTTP Verb
+Create        POST
+Read          GET
+Update        Patch & Put
+Delete        Delete
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Post, Get, Patch, Put, Delete
 
   Researched answer:
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: a parent and a child
 
-  Researched answer:
+  Researched answer: there is  a smart and a dumb component. maybe DIV?
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow